code/modules/mob/living/death.dm

```diff
@@ -1,7 +1,10 @@
-/mob/living/gib(no_brain, no_organs, no_bodyparts)
+/mob/living/gib(no_brain, no_organs, no_bodyparts, explosion = false)

```

Comment:
I still disagree with gib having anything to do with explosions, it should be handled by the `ex_act` itself and not gib.